public/javascripts/starred.js

```diff
@@ -0,0 +1,25 @@
+const populateHUD = () => {
+  $( '.starred-view__container' ).empty()
+
+  const page = ( id, title ) => 
+    $( '<a>', { href: `/items/${id}`, class: 'starred-view__page' })
+      .append( '<div>', { class: 'satrred-view__inner-container' }, `${title}` )
+
+  fetch( '/items/starred', fetchParams( 'get' ) )
+   .then( result => result.json() )
+   .then( ({ data }) => $( '.starred-view__container' ).append(
+     data.map( item => page( item.id, item.title ) )
+   ))
+}
+
+$( document ).ready( () => {
+
+  $( '#star-toggle' ).click( event => {
+    event.preventDefault()
+
+    populateHUD()
+    if($( '.starred-view__HUD' ).css( 'opacity' ) == 0 ) 
```

Comment:
Please always use threequals `===`.  Also, please always use braces to delineate blocks, even when the body of the block is only one line.